spec/lib/spec/lib/user_comm_screener_spec.rb

```diff
@@ -0,0 +1,137 @@
+# frozen_string_literal: true
```

Comment:
The file path here is a little off. I think we meant for this to be `spec/lib/user_comm_screener_spec.rb`?

Reply:
Oh yeah weird, will fix